client/components/password-change/password-change.js

```diff
@@ -0,0 +1,124 @@
+import "./index.css";
+import React from "react";
+import axios from "axios";
+import qs from "qs";
+import PropTypes from "prop-types";
+
+import {passwordConfirmError, passwordChangeApiUrl} from "../../constants";
+
+import getText from "../../utils/get-text";
+
+export default class PasswordChange extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      newPassword1: "",
+      newPassword2: "",
+      errors: {},
+    };
+
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleSubmit(e) {
+    if (e) e.preventDefault();
+    const {orgSlug} = this.props;
+    const url = passwordChangeApiUrl.replace("{orgSlug}", orgSlug);
+    const {newPassword1, newPassword2} = this.state;
+
+    if (newPassword1 !== newPassword2) {
+      this.setState({
+        errors: {
+          newPassword2: passwordConfirmError,
+        },
+      });
+      return null;
+    }
+
+    return axios({
+      method: "post",
+      headers: {
+        "content-type": "application/x-www-form-urlencoded",
+      },
+      url,
+      data: qs.stringify({
+        newPassword1,
+        newPassword2,
+      }),
+    }).catch(error => console.log(error));
+  }
+
+  handleChange(e) {
+    this.setState({[e.target.name]: e.target.value});
+  }
+
+  render() {
+    const {language, passwordChange} = this.props;
+    const {errors} = this.state;
+    return (
+      <div className="owisp-passchange-page">
+        <div className="owisp-passchange-container">
+          <div className="title">{getText(passwordChange.title, language)}</div>
+          <input
+            className="owisp-passchange-input"
+            style={{marginTop: "1rem"}}
+            type="password"
+            name="newPassword1"
+            placeholder={getText(
+              passwordChange.input_fields.password1.placeholder,
+              language,
+            )}
+            onChange={e => this.handleChange(e)}
+          />
+          <input
+            className="owisp-passchange-input"
+            type="password"
+            name="newPassword2"
+            placeholder={getText(
+              passwordChange.input_fields.password2.placeholder,
+              language,
+            )}
+            onChange={e => this.handleChange(e)}
+          />
+          {errors.password2 ? errors.password2 : null}
```

Comment:
You have set error for `newPassword2` key and are checking for `password2` which is always false. Please fix this and wrap the error message in div and style it. [Here](https://github.com/openwisp/openwisp-wifi-login-pages/blob/3402215b3603982ac06d0963d9b5474763883869/client/components/registration/registration.js#L263) is an error message Div. You should make it similar to that one.